src/core/coap/coap_secure.hpp

```diff
@@ -226,8 +317,36 @@ class CoapSecure : public CoapBase
     void *            mTransportContext;
     Message *         mTransmitMessage;
     Tasklet           mTransmitTask;
+
+    bool mLayerTwoSecurity;
```

Comment:
nit: prefer using `mLayerTwoSecurity : 1` to specify bit field.

Reply:
done